### DISCUSSION TITLE
How big is a lotus full-node? (How much SSD should I buy?)

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [filecoin-project/community#603] [Developers]
I have a full-node running but I haven't pruned it in a while, wondering how much space I could save.

Also, the docs say [you'll need enough SSD to run a full node](https://lotus.filecoin.io/lotus/install/prerequisites/#minimal-requirements) - how much is that?

**Comment by web3jenks** [comment]
Hey @PatrickAlphaC , if you are looking to build a full history node, then the current size of the full snapshot is around 2.4TiB. 

Latest snapshot: 
https://lilium.sh/data/dumps/

The Lotus team is working on [splitsotre](https://lotus.filecoin.io/lotus/configure/splitstore/) right now which will allow archival of old chain data for controlling the storage size.

**Reply by PatrickAlphaC** [reply]
Thanks!
